File: Python/백준/Bronze/32801. Generalized FizzBuzz/Generalized FizzBuzz.py
n,a,b = map(int,input().split())
cnt1 = 0; cnt2 = 0; cnt3 = 0 #각각 피즈,버즈,피즈버즈
for i in range(1,n+1):
    if i%a==0 and i%b==0:
        cnt3+=1
    elif i%a==0:
        cnt1+=1
    elif i%b==0:
        cnt2+=1
print(cnt1,cnt2,cnt3)

File: C#/백준/Bronze/32801. Generalized FizzBuzz/Generalized FizzBuzz.cs
using System;

namespace q{
    class w{
        static void Main(){
            int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int cnt1=0,cnt2=0,cnt3=0;
            for (int i=1; i<aa[0]+1; i++){
                if(i%aa[1]==0 && i%aa[2]==0){cnt3++;}
                else if(i%aa[1]==0){cnt1++;}
                else if(i%aa[2]==0){cnt2++;}
            }
            Console.WriteLine($"{cnt1} {cnt2} {cnt3}");
        }
    }
}